pvlib/test/test_pvsystem.py

```diff
@@ -174,6 +174,23 @@ def cec_module_params(sam_data):
     return module_parameters
 
 
+@pytest.fixture(scope="session")
```

Comment:
not a big deal, but if we're still making changes this should not have ``scope="session"``

Reply:
I copied that.  `scope="module"`?

Reply:
no scope argument. ``pytest.fixture`` and nothing else

Reply:
The reasoning for this is that this function returns a mutable dictionary. In principle, one test could inadvertently modify the dictionary, and a second test could pass or fail because of that modification. ``@pytest.fixture`` will call the function and generate a new dictionary each time the data is needed. Adding ``scope="module"`` or ``scope="session"`` instructs ``pytest`` to recreate the data only once per test module (the fixture could be imported into a new module and data would be regenerated) or once per test session (this defeats a large part of the purpose of fixtures).

Reply:
OK. I'll remove the other instances of `scope=session` from fixtures for CEC and SAPM module parameters, but leave the instance that uses `pvsystem.retrieve_sam`